### ISSUE TITLE
App Crash with accounts that possibly have wrong profile photo format?

### ISSUE THREAD

**Issue by sharibarboza** [CMPUT301F17T29/HabitUp#194]
Discovered this bug when I tried to go to the Find a User screen and it crashed the app. I found that by commenting out this line 

```
if (result.getPhoto() != null) {
  resultPhoto.setImageBitmap(result.getPhoto());
}
```

it didn't crash the app anymore so it must have something to do with profile photos.
When I tried to login to the other 2 user accounts, it crashes upon login. Could this might be because a non-webcam photo was added to the user's set profile photo?

**Comment by gojeffcho** [comment]
Yes, I created this bug with my demo accounts and cheat photo method - I have this fixed and worked around in my demo build but haven't committed it because it's single use. Do you want me to make a fix change to not have it break for normal use while I'm doing the video stuff with these accounts?

**Comment by sharibarboza** [comment]
No, it's fine. I'll just leave the thing commented out for now in case I need to go to Find a User. When you're done the video and remove it though, I'll just have to remember to uncomment it later.
